packages/now-next/src/utils.ts

```diff
@@ -12,8 +12,9 @@ import {
   streamToBuffer,
   Lambda,
   isSymbolicLink,
+  NowRewrite,
```

Comment:
This is strange... `NowRewrite` should be imported from `@now/routing-utils`

Reply:
Oh interesting, must of auto-imported from there instead. Updated